src/Language/PureScript/Interactive/Message.hs

```diff
@@ -28,27 +32,20 @@ helpMessage = "The following commands are available:\n\n    " ++
 
   extraHelp =
     "Further information is available on the PureScript documentation repository:\n" ++
-    " --> https://github.com/purescript/documentation/blob/master/PSCi.md"
+    " --> " ++ guideURL
 
 -- | The welcome prologue.
 prologueMessage :: String
 prologueMessage = unlines
-  [ "PSCi, version " ++ showVersion Paths.version
+  [ "PureScript REPL, version " ++ showVersion Paths.version
   , "Type :? for help"
   ]
 
 supportModuleMessage :: String
 supportModuleMessage = unlines
-  [ "PSCi requires the psci-support package to be installed."
-  , "You can install it using Bower as follows:"
-  , ""
-  , "  bower i purescript-psci-support --save-dev"
-  , ""
-  , "Or using psc-package:"
-  , ""
-  , "  psc-package install psci-support"
-  , ""
-  , "For help getting started, visit https://github.com/purescript/documentation/blob/master/PSCi.md"
+  [ "purs repl: No input files, or no psci-support loaded; try running `pulp psci` instead."
```

Comment:
It's straightforward enough to distinguish these two cases that I think we should be doing so. I'd rather leave the `supportModuleMessage` more or less as it is and define a new `noInputMessage` similar to what you have here, but which is only displayed when no input files are provided.

The place to do that seems to be here: https://github.com/noraesae/purescript/blob/005a84f031e975224a31564309296134bbe1a4ad/app/Command/REPL.hs#L313-L315